v2/support/secure_settings_sample_app/assets/iframe.html

```diff
@@ -3,35 +3,47 @@
   <meta charset="utf-8">
   <!-- http://garden.zendesk.com -->
   <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@zendeskgarden/css-bedrock@7.0" type="text/css">
+  <link rel="stylesheet" href="styles.css" type="text/css">
 </head>
 <body>
-  <img class="cat-picture" height="200" width="250"></h2>
+  <img class="cat-picture" height="200" width="250"></img>
+  <div class="button" id="new-cat-button" height="20" width="50">Adopt a new cat</div>
   <!-- https://github.com/zendesk/zendesk_app_framework_sdk -->
   <script type="text/javascript" src="https://assets.zendesk.com/apps/sdk/2.0/zaf_sdk.js"></script>
   <script>
     // Initialise the Zendesk JavaScript API client
     // https://developer.zendesk.com/apps/docs/apps-v2
```

Comment:
You may want to update this link to https://developer.zendesk.com/apps/docs/developer-guide/getting_started

As the one you have in this comment 404s